moveit_ros/occupancy_map_monitor/src/occupancy_map_server.cpp

```diff
@@ -75,8 +75,8 @@ int main(int argc, char** argv)
 {
   rclcpp::init(argc, argv);
   auto node = rclcpp::Node::make_shared("occupancy_map_server");
-  auto octree_binary_pub =
-      node->create_publisher<octomap_msgs::msg::Octomap>("octomap_binary", RMW_QOS_POLICY_HISTORY_KEEP_LAST);
+  rclcpp::QoS qos(rclcpp::QoSInitialization::from_rmw(rmw_qos_profile_default));
+  auto octree_binary_pub = node->create_publisher<octomap_msgs::msg::Octomap>("octomap_binary", qos);
```

Comment:
Consider writing this like this:
`auto octree_binary_pub = node->create_publisher<octomap_msgs::msg::Octomap>("octomap_binary", rclcpp::SystemDefaultsQoS());`

... and removing the `qos` temporary variable.